src/components/submission/feed-entry.vue

```diff
@@ -35,9 +35,12 @@ except according to the terms contained in the LICENSE file.
         <span class="icon-magic-wand entity-icon"></span>
         <i18n-t keypath="title.entity.create">
           <template #label>
-            <router-link :to="entityPath(projectId, entityDataset(entry), entityUuid(entry))">
+            <router-link v-if="entityLabel(entry) != null" :to="entityPath(projectId, entityDataset(entry), entityUuid(entry))">
```

Comment:
I think we can use optional chaining here and entityLabel function is not required at all:

`entry.details?.entity?. currentVersion?. label`